code/api/src/modules/subscription/model.js

```diff
@@ -9,12 +11,26 @@ module.exports = function(sequelize, DataTypes) {
     crateId: {
       type: DataTypes.INTEGER
     }
-  })
+  }, {
+      hooks: {
+        afterCreate: async (subscription) => {
+          let user = subscription.dataValues.userId;
+          let subscriptionId = subscription.dataValues.id;
+          let date = new Date(new Date().getTime()+(5*24*60*60*1000))
+
+          for(let i = 1; i < 13; i++){
```

Comment:
I think this is a good solution for the time being, we can circle back if we have time!